client/blocks/get-apps/desktop-download-card.jsx

```diff
@@ -153,22 +229,122 @@ class DesktopDownloadCard extends Component {
 
 		switch ( platformLink ) {
 			case MAC_LINK:
-				return <a href={ localizeUrl( platformLink ) } onClick={ trackMacClick } />;
+				return (
+					<a
+						href={ localizeUrl( platformLink ) }
+						onClick={ trackMacClick }
+						className={ classnames( {
+							'get-apps__desktop-link': displayJetpackAppBranding,
+						} ) }
+					/>
+				);
 			case LINUX_TAR_LINK:
-				return <a href={ localizeUrl( platformLink ) } onClick={ trackLinuxTarClick } />;
+				return (
+					<a
+						href={ localizeUrl( platformLink ) }
+						onClick={ trackLinuxTarClick }
+						className={ classnames( {
+							'get-apps__desktop-link': displayJetpackAppBranding,
+						} ) }
+					/>
+				);
 			case LINUX_DEB_LINK:
-				return <a href={ localizeUrl( platformLink ) } onClick={ trackLinuxDebClick } />;
+				return (
+					<a
+						href={ localizeUrl( platformLink ) }
+						onClick={ trackLinuxDebClick }
+						className={ classnames( {
+							'get-apps__desktop-link': displayJetpackAppBranding,
+						} ) }
+					/>
+				);
 			default:
-				return <a href={ localizeUrl( platformLink ) } onClick={ trackWindowsClick } />;
+				return (
+					<a
+						href={ localizeUrl( platformLink ) }
+						onClick={ trackWindowsClick }
+						className={ classnames( {
+							'get-apps__desktop-link': displayJetpackAppBranding,
+						} ) }
+					/>
+				);
 		}
 	}
 
-	render() {
-		const { translate } = this.props;
+	getMobileDeviceDownloadOptions() {
+		return translate( 'Visit {{a}}desktop.wordpress.com{{/a}} on your desktop.', {
+			components: {
+				a: <a href="https://desktop.wordpress.com" className="get-apps__desktop-link" />,
+			},
+		} );
+	}
+
+	getDesktopDeviceDownloadOptions() {
+		const platform =
+			navigator.platform && navigator.platform.length > 0 ? navigator.platform : false;
+
+		return (
+			<>
+				<Button
+					className="get-apps__desktop-button jetpack"
+					href={ localizeUrl( this.getButtonLink( platform ) ) }
+					onClick={ this.getButtonClickHandler( platform ) }
+				>
+					{ this.getButtonText( platform ) }
+				</Button>
+				<div className="get-apps__requirements-wrapper">
+					{ this.getPlatformImage( platform ) }
+					<p className="get-apps__requirements-paragraph">
+						{ this.getRequirementsText( platform ) }
+					</p>
+				</div>
+				<div className="get-apps__also-available jetpack">
+					<p>{ translate( 'Also available for:' ) }</p>
+					<div className="get-apps__also-available-link-group">
+						{ this.getAlsoAvailableTextJetpack( platform ) }
+					</div>
+				</div>
+			</>
+		);
+	}
+
+	getJetpackBrandedPanel() {
+		const { isMobile } = userAgent;
+
+		return (
+			<>
+				<div className="get-apps__card-text jetpack">
+					<SVGIcon
+						name="desktop-app-logo"
```

Comment:
Similar to my other comment, I think it makes sense to hide this from screen readers. 
```suggestion
						aria-hidden="true"
						name="desktop-app-logo"
```

Reply:
Implemented in https://github.com/Automattic/wp-calypso/pull/69674/commits/ddb0f0c17772d96967669a39cedc2eba2d06a89e, thank you!